docs/changes.rst

```diff
@@ -14,6 +14,7 @@ Pending
   memory leaks and sometimes very verbose and hard to silence output in some
   environments (but not others). The maintainers judged that time and effort is
   better invested elsewhere.
+* Added support for psycopg3 3.
```

Comment:
```suggestion
* Added support for psycopg3 3.
```

I'm not sure what's better, but this should probably be either "psycopg3" or "psycopg 3", not "psycopg3 3"

Reply:
That one is on me. I changed it to get past the spellcheck originally. I must not have cleaned up after myself properly.